src/plugins/intel_gna/src/optimizer/gna_pass_manager.cpp

```diff
@@ -2436,6 +2506,85 @@ void TransposeWeightsFromNCHWToNHWCPass::run() {
     }
 }
 
+void FuseFullyConnectedWithEltwisePass::run() {
+    // This legacy pass removes the Eltwise (only if it performs SUM op) from between FC and Any.
+    // The Const layer blob data is added to biases blob data of FC layer.
+    // Finally Const and Reshape (if exists) are also removed
+    //
+    // This operation can be illustrated as follows:
+    //
+    // --
+    //  Original:               Result:         Removed:
+    //
+    //    FC                     FC              (Eltwise)
+    //    |                      |               (Const)
+    //  Reshape                 Any              (Reshape)
+    // (optional)  Const
+    //    |       /
+    //  Eltwise(sum)
+    //    |
+    //  Any (e.g. ReLU)
```

Comment:
Can we have ReLU at this point? Or will we already have PWLApproximation?

Reply:
Yes, ReLU remains at this point. It's not touched by PWLApproximation.